src/core/urdb.jl

```diff
@@ -359,45 +342,27 @@ end
 
 
 """
-    parse_urdb_demand_tiers(A::Array; bigM=1.0e8)
+get_num_tiers(d::Dict)
 
-set up and validate demand tiers
-    returns demand_tiers::Array{Float64, n_tiers}
+    get maximum number of demand tiers in any period from scrubbed demand rate structure
```

Comment:
This function is used for energy rates as well but the description says just demand

Reply:
Yes it should; this snippet of code is actually redundant because it all occurs in the scrubbing function to make the number of tiers equal across periods.  I refactored the code so we don't do the same thing twice here:  https://github.com/NREL/REopt.jl/commit/5fbc355766cf2dad01394c4a16c7006852a68d19